tests/basics/gen_yield_from.py.exp

```diff
@@ -1,14 +0,0 @@
-here1
```

Comment:
Deleting this and other files doesn't mean that the tests will fail with CPython3.4?

Reply:
No, these tests should now test the subset of generator functionality that is the same in CPython 3.4-3.7.  All tests that differ on 3.4 vs 3.5 are moved to generator_pep479.py test.

Reply:
Note that Travis uses CPython 3.4.3 and it succeeds ok here.